### PR TITLE
Feature local UI

### PR DESCRIPTION
**Pull request by shreyas-damle**
Feature Local UI : https://github.com/daxa-ai/pebblo/issues/105

This includes frontend as well as backend changes for local UI.

### CODE REVIEW

**Review by KunalJadhav5 — APPROVED** [review]
LGTM

**Review by KumarNitin19 — APPROVED** [review]
Looks good to me!

**Comment by Raj725** [comment]
@KumarNitin19
I tried to run Pebblo and access the local UI. It works properly when executed from the project directory, but It throws an error when running Pebblo outside of the project directory.

```
INFO:     ::1:58552 - "GET /pebblo/ HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback  
...
...
jinja2.exceptions.TemplateNotFound: index.html
```

File: pebblo/app/routers/local_ui_routers.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# Add routes to the class-based router
+local_ui_router_instance.router.add_api_route(
+    "/", App.dashboard, methods=["GET"], response_class=HTMLResponse
+)
+local_ui_router_instance.router.add_api_route(
+    "/app/", App.appDetails, methods=["GET"], response_class=HTMLResponse
+)
+local_ui_router_instance.router.add_api_route(
+    "/report/", App.getReport, methods=["GET"], response_class=dict
```
Comment (Raj725) [inline pebblo/app/routers/local_ui_routers.py:15]: Use the correct response class

File: pebblo/reports/pebblo_report_xhtml2pdf.pdf
Comment (Raj725) [inline pebblo/reports/pebblo_report_xhtml2pdf.pdf:1]: Is this file needed in the source code?

File: pebblo/app/routers/local_ui_routers.py (hunk 1 above)
Comment (Raj725) [inline pebblo/app/routers/local_ui_routers.py:15]: Use the correct response class

Comment (KumarNitin19) [reply pebblo/app/routers/local_ui_routers.py:15]: @Raj725 comment resolved in PR: https://github.com/daxa-ai/pebblo/pull/232/files

File: pebblo/reports/pebblo_report_xhtml2pdf.pdf
Comment (KumarNitin19) [reply pebblo/reports/pebblo_report_xhtml2pdf.pdf:1]: @Raj725 No we don't need that.

**Review by rutujaac — APPROVED** [review]
LGTM

**Review by Raj725 — APPROVED** [review]
Backend changes LGTM
